pkg/downloader/manager.go

```diff
@@ -206,6 +206,7 @@ func (m *Manager) downloadAll(deps []*chartutil.Dependency) error {
 	}
 
 	destPath := filepath.Join(m.ChartPath, "charts")
+	tmpPath := filepath.Join(m.ChartPath, "tmpcharts")
```

Comment:
minor nitpick, but a common pattern would be to name the directory `charts.tmp`. I wouldn't consider this a blocker, but just a passing note :)